src/Makefile

```diff
@@ -79,7 +79,7 @@ endif
 
 RT_LLVM_LIBS := support
 
-ifeq ($(LLVM_VER_MAJ),16)
+ifeq ($(shell test $(LLVM_VER_MAJ) -ge 16),1)
```

Comment:
On this branch I'm getting
```console
% make -C src print-RT_LLVM_LIBS
RT_LLVM_LIBS=support
```
I don't think this is working.  Maybe do https://github.com/JuliaLang/julia/pull/53083#discussion_r1469963998?

Reply:
I managed to fix it by adding `&& echo true`.

Reply:
Note that #53083 was merged and that was working for me with both llvm 16 and 17. You now need to rebase on master because of the conflicts.

Reply:
Ah damm. Okay.